### PR TITLE
Lock command strings to prevent issues with translations

### LINKED ISSUE #172663: Handle commands in localized string better
image

This likely doesn't repro in most other languages because those are human reviewed and there is a comment about it, but just in case, it's best to just either lock it by adding a:
```
{Locked='](command:....)'}
```
comment or just removing the link syntax and just have the translators see the contents of the button.

code pointer: https://github.com/microsoft/vscode/blob/0febc5483ff43cbf730118ec59eb623e2a205500/src/vs/workbench/contrib/debug/browser/welcomeView.ts#L108-L144

### PR DESCRIPTION
**Pull request by TylerLeonhardt**
Fixes https://github.com/microsoft/vscode/issues/172663

### CODE REVIEW

File: src/vs/workbench/contrib/debug/browser/welcomeView.ts [hunk 1]
```diff
@@ -107,37 +107,75 @@ export class WelcomeView extends ViewPane {
...
-		"[Open a file](command:{0}) which can be debugged or run.", (isMacintosh && !isWeb) ? OpenFileFolderAction.ID : OpenFileAction.ID),
+	content: localize(
+		{
+			key: 'openAFileWhichCanBeDebugged',
+			comment: [
+				'Please do not translate the word "command", it is part of our internal syntax which must not change',
+				'{Locked="](command:{0})"}'
+			]
+		},
+		"[Open a file](command:{0}) which can be debugged or run.", (isMacintosh && !isWeb) ? OpenFileFolderAction.ID : OpenFileAction.ID
```
Comment (Tyriar) [inline src/vs/workbench/contrib/debug/browser/welcomeView.ts:118]: It's a little more complicated, but the better way to do this imo is something like this:

```ts
localize('', '{0} which can be debugged or run', `[${localize('', 'Open a file')}](command:${(isMacintosh && !isWeb) ? OpenFileFolderAction.ID : OpenFileAction.ID})`)
```

That prevents the translator potentially messing up.

Comment (TylerLeonhardt) [reply src/vs/workbench/contrib/debug/browser/welcomeView.ts:118]: I disagree. The sentence is `Open a file which can be debugged or run`. Breaking it up could mean that `Open a file` gets translated in a weird way that doesn't flow with `which can be debugged or run`.

Comment (TylerLeonhardt) [reply src/vs/workbench/contrib/debug/browser/welcomeView.ts:118]: Example:
image
vs
image

Comment (Tyriar) [reply src/vs/workbench/contrib/debug/browser/welcomeView.ts:118]: That's true. How about putting markers instead and explaining in the command it needs to wrap the text?

```ts
"{0}Open a file{1} which can be debugged or run."
```

Comment (TylerLeonhardt) [reply src/vs/workbench/contrib/debug/browser/welcomeView.ts:118]: That would also work.

**Review by roblourens — APPROVED** [review]

**Pull request merged**